code/app/src/main/java/com/example/sigma_blue/fragments/EditFragment.java

```diff
@@ -136,6 +151,32 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState)
         SimpleDateFormat sdf = new SimpleDateFormat(getResources().getString(R.string.date_format));
         textDate.setText(sdf.format(currentItem.getDate()));
 
+        //ITEM IMAGE RELATED CHANGES
+        // trying to get the path of image, and put it on the add item
+        String tempImagePath = globalContext.getCurrentItem().getPhotoPath();
```

Comment:
Code here looks like a copy of the details view fragment. Should be factored out to an inherited/static function for code reuse.